apps/settings/src/components/Users/NewUserModal.vue

```diff
@@ -100,7 +100,7 @@
 				class="modal__item">
 				<label class="modal__label"
 					for="new-user-sub-admin">
-					{{ t('settings', 'Administered groups') }}
+					{{ t('settings', 'Manages user(s) in the following group(s)') }}
```

Comment:
I think the current wording is
```suggestion
					{{ t('settings', 'Manages accounts in the following group(s)') }}
```